### DISCUSSION TITLE
Can we create a single cron schedule to run for a different date and different times

### DISCUSSION THREAD

**Discussion by mahesmart16** [quartz-scheduler/quartz#1259] [Q&A] [answered]
Hi Everyone,

I am using Databricks workflows to schedule a job and it used quartz cron schedule. My requirement is to run a single workflow or job that should run on monday 2 AM and tuesday 4:30 AM,Is there any feasibility to achieve this. I tried testing this my using the cron schedule but it is running monday @ 2:00 and 4:30 am and also on tuesday @ 2:00 and 4:30 am, Please advise.

**Comment by jhouserizer** [comment, marked as answer]
Hi. There is no way to do this with a single trigger (chron schedule).  The intended way to do this is to associate multiple triggers with the same job (in your case one trigger for monday and another trigger for tuesday).
